portfolio/src/main/webapp/index.html

```diff
@@ -1,19 +1,144 @@
+                    </li>
+                </ul>
+
+                <li><h3>Code: </h3></li>
+
+                <ul>
+                    <li> <b> My Portfolio:</b> Using basic knowlegde about HTML, CSS & JavaScript, I coded this web page
+                        for <a href="https://buildyourfuture.withgoogle.com/programs/softwareproductsprint/">Software Product Sprint by Google</a>>.
+                    </li>
+                    
+                    <li> <b> Calculator:</b> Using basic knowledge about JAVA, 3 classmates and me coded a calculator capable of performing basic operations
+                        implements a postfixed algorithm and stacks.
+                    </li>
+                </ul>
+
+            </ul>
+        </section>
+
+        <section>
+            <h1>My Portfolio</h1>
+            <p>This is my portfolio.</p>
+            <p>Click <a href="/hello">here</a> to see content created by server code.</p>
+            <p>Click here to get a random message:</p>
+            <button onclick="addRandomGreeting()">Hello</button>
+            <div id="greeting-container"></div>
+            
+            <h2>Random Facts About Me</h2>
+            <p> Function not available </p>
+        </section>
+
+        <section>
+            <h2>Awards</h2>
+            <ul>
+                <li><b> Bronze medal</b> (2018) and <b>Honorific Mention</b> (2019) in 
+                Mexican Mathematical Olympiad - 
+                <a href= "https://www.ommenlinea.org"> <abbr title="Olimpiada Mexicana de Matemáticas"> OMM </abbr> </a> </li>
+
+                <li><b> 2 Bronze medals </b> (2018 & 2019) (2019) in Regional Northeast Mathematical Olympiad </li>
+
+                <li><b> Gold medal </b> (2020) in Regional Math Contest of Private School  </li>
+
+                <li><b> Bronze medal </b> (2020) in National Online Math Contest of Private School  </li>
+
+                <li><b> Two time champion </b> (2018 & 2020) in Regional Contest of Science of the Technological Institute of Mexico  </li>
+
+                <li><b> One of the 100 winners </b> (2014) in Knowlegde Olympiad for Youth   </li>
+            </ul>
+        </section>
+
+        <section>
+            <h2>Skills</h2>
+            <ul>
+                <li> <b> Programming Language: </b> Java, MATLAB & C++</li>
+                <li> <b> Language: </b> Spanish(Native), English (Intermediate)</li>
+                <li> <b> General Problem Solving: </b> Solving mathematical problems and beginner in competitive programming</li>
+            </ul>
+        </section>
+
+        <section>
+            <h2>Soft skills</h2>
+            <ul>
+                <li> <figure>
+                    <img src= decisiveness.png alt="decisive by Nithinan Tatah from the Noun Project" style = "width:50px; height:50px">
```

Comment:
nit: prefer to always use quotes, e.g. `src="decisiveness.png"` as some image names won't work without quotes